src/main/java/com/iota/iri/service/tipselection/impl/RecursiveWeightCalculator.java

```diff
@@ -55,53 +53,49 @@ public RecursiveWeightCalculator(Tangle tangle, SnapshotProvider snapshotProvide
 
         // Estimated capacity per depth, assumes 5 minute gap in between milestones, at 3tps
         UnIterableMap<HashId, Integer> hashWeight = createTxHashToCumulativeWeightMap( 5 * 60 * 3 * depth);
```

Comment:
Maybe call it `hashToWeight` or `hashWeightMap`.
This variable has an integer name